docs/changes/16628.other.rst

```diff
@@ -0,0 +1,4 @@
+Explicitly set usedforsecurity=False when using hashlib.md5. Without this, hashlib.md5 will be blocked in FIPS mode.
+FIPS (Federal Information Processing Standards) is a set of standards created by NIST (National Institute of Standards and Technology) for US government agencies regarding computer security and interoperability.
+Read more here: https://en.wikipedia.org/wiki/Federal_Information_Processing_Standards
+
```

Comment:
Let's avoid "live" links so we do not have to go back and change history in the future. I think the text without link is sufficient.
```suggestion
```

Reply:
08b60e30eae0f0395186ae8ecc04c3778b648a04